toolchain/check/handle_alias.cpp

```diff
@@ -16,11 +16,15 @@ auto HandleParseNode(Context& context, Parse::AliasIntroducerId /*node_id*/)
     -> bool {
   context.decl_introducer_state_stack().Push<Lex::TokenKind::Alias>();
   context.decl_name_stack().PushScopeAndStartName();
+  context.pattern_block_stack().Push();
   return true;
 }
 
-auto HandleParseNode(Context& /*context*/,
-                     Parse::AliasInitializerId /*node_id*/) -> bool {
+auto HandleParseNode(Context& context, Parse::AliasInitializerId /*node_id*/)
+    -> bool {
+  // TODO: when/if parameterized aliases are supported, this must be
+  // attached to the `BindAlias` inst.
+  (void)context.pattern_block_stack().Pop();
```

Comment:
```suggestion
  context.pattern_block_stack().PopAndDiscard();
```

Generally I don't think we write `(void)` except where required by the compiler for an unused variable/parameter. If you're trying to indicate that there's a value being discarded, maybe the `PopAndDiscard` call would capture that?

(note, this same comment would apply elsewhere that you're doing this)